### PR TITLE
64 calendar into react

### PR DESCRIPTION
**Pull request by noobiba**
Transformed course schedule into React component. 
@lauralindal @sarahkami Please review pull request. Thank you! :-)

### CODE REVIEW

File: src/Weekday.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+class Weekday extends React.Component {
+  render() {
+    return (
+      <div>
+      Bananenkuchen
```
Comment (lauralindal) [inline src/Weekday.js:7]: ? I love banana bread... but what is it doing here? ;D

**Review by lauralindal — CHANGES_REQUESTED** [review]
I think this PR has a lot of (older) commits that don't belong in there, maybe this also causes the merge conflict. @noobiba please fix merge conflicts <3

**Comment by ghost** [comment]
I think it might be best to ignore the pull request for the moment until we have something that works. Initially we just put the html in a single React Component, but I think Tanja pushed  another commit after we played around with it some more. I think the weekly schedule can stay as html in the index for the moment, since I see no benefit to adding something to the dev that is not working yet.

**Comment by noobiba** [comment]
I compared both pull requests (#70 add-legend and #64 calendar-into react) and I see, that there are a lot of commits from my other branch in here. I prefer deleting this pull request. We did only some trying. Nothing important in here. I thought Ola an me fixed this commit mess that I started pushing one commit to the wrong branch. Sorry.

**Pull request closed without merging**